loader/CMakeLists.txt

```diff
@@ -298,8 +298,10 @@ else()
         if(PKG_CONFIG_FOUND)
             set(VK_API_VERSION "${VulkanHeaders_VERSION_MAJOR}.${VulkanHeaders_VERSION_MINOR}.${VulkanHeaders_VERSION_PATCH}")
             foreach(LIB ${CMAKE_CXX_IMPLICIT_LINK_LIBRARIES} ${PLATFORM_LIBS})
-                set(PRIVATE_LIBS "${PRIVATE_LIBS} -l${LIB}")
+                list(APPEND PRIVATE_LIBS_LIST "-l${LIB}")
             endforeach()
+            list(REMOVE_DUPLICATES PRIVATE_LIBS_LIST)
+            list(JOIN PRIVATE_LIBS_LIST " " PRIVATE_LIBS)
```

Comment:
I'm not a CMake expert, so if there's a cleverer way to prevent adding duplicates to a string without having to use a list and a `REMOVE_DUPLICATES` filter, that could be used it.

Or maybe iterating over both `CMAKE_CXX_IMPLICIT_LINK_LIBRARIES` and `PLATFORM_LIBS` is overkill, I did not investigate further.